src/kernel/tty.zig

```diff
@@ -1,13 +1,8 @@
-// Zig version: 0.4.0
+const is_test = @import("builtin").is_test;
+const fmt = std.fmt;
 
-const vga = @import("vga.zig");
-const log = @import("log.zig");
-
-const expectEqual = @import("std").testing.expectEqual;
-const expectError = @import("std").testing.expectError;
-const expect = @import("std").testing.expect;
-const warn = @import("std").debug.warn;
-const fmt = @import("std").fmt;
+const vga = if (is_test) @import("../../test/mock/kernel/vga_mock.zig") else @import("vga.zig");
+const log = if (is_test) @import("../../test/mock/kernel/log_mock.zig") else @import("log.zig");
```

Comment:
One quick thought. If we're testing, could we change the path that files are imported from rather than check them here? That would be more streamlined, if it's possible.

Reply:
I'll have ago, as you can add an include path in the building

Reply:
Nice, it's not a big deal if it doesn't work, just a nice-to-have.